marketplace/netlify-build/.babelrc

```diff
@@ -0,0 +1,48 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": false,
+        "modules": false
+      }
+    ],
+    [
+      "@babel/preset-react",
+      {
+        "useBuiltIns": true
+      }
+    ]
+  ],
+  "plugins": [
+    [
+      "@babel/plugin-proposal-class-properties",
+      {
+        "loose": true
+      }
+    ],
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "corejs": false,
+        "helpers": false,
+        "regenerator": true
+      }
+    ]
+  ],
+  "env": {
+    "test": {
```

Comment:
not sure if we need it with a new jest setup

Reply:
I'll take a look

Reply:
Ok, removed.